src/zope/app/apidoc/utilities.py

```diff
@@ -325,10 +366,10 @@ def renderText(text, module=None, format=None, dedent=True):
 
     assert format in _format_dict.values()
 
-    text = dedentString(text)
-
-    if not isinstance(text, unicode):
+    if isinstance(text, bytes):
         text = text.decode('latin-1', 'replace')
```

Comment:
The charset customarily used on Windows is cp1252, which differs from latin-1 in various subtle ways (mostly in punctuation, IIRC).

I would expect source code of Zope-ish things to be in UTF-8, if we're talking about open source packages installed from PyPI.  If we're talking about homegrown code of random Zope instances, that's a different question, but Latin-1 seems very unfriendly and English-centered.

(I notice such things since some Lithuanian letters are not present in Latin-1.)